core/consensus/consensus_event.hpp

```diff
@@ -11,11 +11,12 @@ namespace consensus_event {
 
 struct ConsensusEvent {
     std::unique_ptr<abstract_transaction::AbstractTransaction> tx;
-    std::vector<std::string> txSignatures;
+    std::vector<std::string> signatures;
     std::string merkleRoot;
+    unsigned long long order = 0;
     std::vector<std::string> merkleRootSignatures;
 
-    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> tx){
+    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> tx) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Struct 'ConsensusEvent' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4256066295/issues/source?bid=3712501&fileBranchId=3831026#l19)